### DISCUSSION TITLE
Error 42 crashing the rig on dual mining ETH+TON

### DISCUSSION THREAD

**Discussion by aminlv** [Lolliedieb/lolMiner-releases#1042] [Q&A] [answered]
i tried it few times with my RX 580 8GB  after 30 minutes or so i get error 42 in windows 10 system crash is so sever i had to unplug the power cord of my PC to make system work again!

is there anyway to fix this issue whats error 42 anyway?

**Comment by eaamgh** [comment]
Same thing. Not sure if error code is 42 but lolminer crashes right after DAG verification. I was also getting some message about "not receiving authorization or job after 300 seconds". Didn't save it, sorry. What's weird is retrying it won't work. Until it does... For no apparent reason, it will start working again...

**Reply by eaamgh** [reply]
Just to add to it, I also have DUALTON running on a couple of HiveOS rigs. Over 12h mining in DUALTON mode and no issues. Seems to be a Windows-related issue. And by the way, zero impact on ETH mining on rx 6800xt and non-LHR RTX3070s.

**Reply by aminlv** [reply]
yeah might be windows issue i check it later win linux see if its working properly or not there

**Reply by eaamgh** [reply]
Try this: switch it back to ETH only mining, leave it running for a couple of minutes and the back to DUALTON mining. Can't tell why, but it will work for some time again

**Comment by aminlv** [comment]
***well update on my part in fresh installed ubuntu Linux dual mining in same system worked flawlessly!

so either windows version have some sorta issue!

**Reply by eaamgh** [reply]
Yep. Fixed on 1.44. Running stable for over 2h now

**Reply by aminlv** [reply]
well lucky u my saphire nitro+ rx 580 8gb still crashes on windows
i tried solo ton mining to see if it stable or not after 4 hours it crashed w.o afterburner & default ROM
so yeah no dual mining for me on amd atm just my few navis i guess...

**Comment by aminlv** [comment, marked as answer]
*** update ***
version 1.45 fixed the crash issue for rx 580 series
have fun farming
